cPP_DSC.adoc

```diff
@@ -356,58 +356,58 @@ One way to categorize keys is by the cryptographic functions they are allowed to
 
 .Example Key Hierarchy
 [[ExampleKeyHierarchy]]
 image::examplekeyhierarchy.png[,600]
 
-Roles may play an important part in key hierarchies. One of the simplest models enforces a different hierarchy for each role at the root key level. Another way to put this is each hierarchy at the root key level supports a different role. However, for more complexity, once intermediate keys are allowed, then each intermediate key could serve as the root of a hierarchy of keys for a different role. Here is where the key functions and the roles come together. Roles may further divide into which role has the right to use a key, which role has the right to move the key from one parent to another, which role has the right to destroy a key, etc.
+Roles may play an important part in key hierarchies. One of the simplest models enforces a different hierarchy for each role at the Root Key level. Another way to put this is each hierarchy at the Root Key level supports a different role. However, for more complexity, once Intermediate Keys are allowed, then each Intermediate Key could serve as the root of a hierarchy of keys for a different role. Here is where the key functions and the roles come together. Roles may further divide into which role has the right to use a key, which role has the right to move the key from one parent to another, which role has the right to destroy a key, etc.
 
 ==== Protected Storage Locations
 
-This cPP covers several different types of storage locations for keys and critical security parameters (CSPs) such as authentication tokens. Some DSCs may have a generous amount of protected storage internal to themselves, which allows it to accommodate all keys and CSPs in operational use, whether the DSC is performing operations to administer itself or operations on behalf of users. Other DSCs may have a minimal amount of protected storage locations with just enough to accommodate root keys along with a limited number of operational keys and CSPs for user authorized sessions.
+This cPP covers several different types of storage locations for Security Data Elements (SDEs) such as keys and authentication tokens. Some DSCs may have a generous amount of protected storage internal to themselves, which allows it to accommodate all SDEs in operational use, whether the DSC is performing operations to administer itself or operations on behalf of users. Other DSCs may have a minimal amount of protected storage locations with just enough to accommodate root keys along with a limited number of operational SDEs for user authorized sessions.
```

Comment:
As mentioned above, I think it would be more coherent to use the established terms for SDEs instead of introducing the CSPs.

Reply:
#263